code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -188,6 +188,18 @@
 /turf/open/floor/plating/asteroid/airless/cave/volcanic/has_data //subtype for producing a tunnel with given data
 	has_data = TRUE
 
+/turf/open/floor/plating/asteroid/airless/cave/planet
+	mob_spawn_list = list(/mob/living/simple_animal/hostile/asteroid/goliath/beast = 10, /mob/living/simple_animal/hostile/spawner/lavaland/goliath = 3, \
+		/mob/living/simple_animal/hostile/asteroid/basilisk/watcher = 8, /mob/living/simple_animal/hostile/spawner/lavaland = 2, \
```

Comment:
you changed the spawnrate of mobs in asteroid turf fuckign calling the police on u make a new pr u cunt